### PR TITLE
Simple forms: Content updates to 21P-0847

### PR DESCRIPTION
**Pull request by gmrabian**
## Summary

- Content updates per CAIA review
- Rearrange form to include deceased claimant name and date of death
- Ask relationship later in form
- Rework tests and pages to match new order

## Related issue(s)

- https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/392

## Testing done

- Viewed content changes locally

## What areas of the site does it impact?
Form 21P-0847 (not prod)

### CODE REVIEW

File: src/applications/simple-forms/21P-0847/pages/deceasedClaimantPersonalInformation.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  fullNameNoSuffixSchema,
+  fullNameNoSuffixUI,
+  titleSchema,
+  titleUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    rjsf: titleUI(
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21P-0847/pages/deceasedClaimantPersonalInformation.js:13]: can you rename this and the other rjsf cases (if it's not too much work) to something else? rjsf was used in the mock form as a contrast to 'webComponent' vs 'rjsf'.  
I noticed in some forms they use 'view:something'. this helps indicate to the backend that they shouldn't look at this field's data, its just for frontend/view purposes.
maybe like title, or deceasedClaimantTitle, or 'view:deceasedClaimantTitle' ?

File: src/applications/simple-forms/21P-0847/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -16,24 +16,24 @@ const childContent = (
...
-      Once a substitution request is granted, the substitute claimant must
-      complete any action required by law or regulation within the time period
-      remaining for the original claimant to take such action on the date of
-      their death. The substitute has the same rights as would have applied to
-      the original claimant had they not died, including the rights regarding
-      hearings, representation, legacy appeals, decision review options for a
-      claim not finally adjudicated, and submission of evidence. The substitue
-      also has the right to request a decision review of a deceased claimant’s
-      claim that is not yet final.
+      After we approve your substitution request you’re responsible for taking
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21P-0847/containers/ConfirmationPage.jsx:19]: maybe a question for content - do you think we should have a comma after "substitution request"? not a blocking issue, just something to ask about.

Comment (gmrabian) [reply src/applications/simple-forms/21P-0847/containers/ConfirmationPage.jsx:19]: @jeana-adhoc ^

File: src/applications/simple-forms/21P-0847/pages/deceasedClaimantPersonalInformation.js (hunk 1 above)
Comment (gmrabian) [reply src/applications/simple-forms/21P-0847/pages/deceasedClaimantPersonalInformation.js]: good callout! done.

**Review by rhasselle-oddball — DISMISSED** [review]

File: src/applications/simple-forms/21P-0847/containers/ConfirmationPage.jsx (hunk 2 above)
Comment (jeana-adhoc) [reply src/applications/simple-forms/21P-0847/containers/ConfirmationPage.jsx:19]: Good eye @rhasselle-oddball. We missed transposing this from the word doc. Yes, it should read: 

`After we approve your substitution request, you're responsible...`

Comment (gmrabian) [reply src/applications/simple-forms/21P-0847/containers/ConfirmationPage.jsx]: Added the comma. Thanks!

**Review by rokubop — APPROVED** [review]

**Review by rhasselle-oddball — APPROVED** [review]

**Pull request merged**